evm/db/account.py

```diff
@@ -403,7 +404,7 @@ def commit(self, changeset: Tuple[UUID, UUID]) -> None:
         self._journaltrie.commit(trie_changeset)
 
     def make_state_root(self) -> Hash32:
-        self.logger.debug("Generating AccountDB trie")
+        self.logger.log(LOG_TRACE, "Generating AccountDB trie")
```

Comment:
See, my comment above. I think that can simple be `self.logger.trace(...)` as we do it in the `computation` class.